docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -139,14 +203,35 @@ You can perform the following steps with these scripts:
 For the most accurate migration assessment, the source database must be actively handling its typical workloads at the time the metadata is gathered. This ensures that the recommendations for sharding strategies and cluster sizing are well-aligned with the database's real-world performance and operational needs.
     {{< /warning >}}
 
-1. View the assessment report.
+1. View the assessment report by configuring the [yugabyted](/preview/reference/configuration/yugabyted/) UI. Use the yugabyted UI to review the assessment report, which includes migration strategies, complexity, and effort estimates.
+    1. Start a local YugabyteDB cluster. Refer to the steps described in [Use a local cluster](/preview/tutorials/quick-start/macos/).
+
+        {{< note title="Note" >}}
+  After a migration assessment, if you choose to migrate using the open source YugabyteDB, you will be using this same local cluster as your [target database](../../introduction/#target-database).
+        {{< /note >}}
 
-    Use the yugabyted UI to review the assessment report, which includes migration strategies, complexity, and effort estimates.
+    1. To see the Voyager migration workflow details in the UI, set the following configuration parameters before starting the migration:
+
+        ```yaml
+        ### Control plane type refers to the deployment type of YugabyteDB
+        control-plane-type: yugabyted
+
+        ### YSQL connection string
+        ### Provide the standard PostgreSQL connection parameters, including user name, host name, and port. For example, postgresql://yugabyte:yugabyte@127.0.0.1:5433
+        yugabyted-db-conn-string: postgresql://yugabyte:yugabyte@127.0.0.1:5433
+        ```
+
+        Provide the standard PostgreSQL connection parameters, including user name, host name, and port. For example, `postgresql://yugabyte:yugabyte@127.0.0.1:5433`
+
+        {{< note title="Note" >}}
+
+  Don't include the `dbname` parameter in the connection string; the default `yugabyte` database is used to store the meta information for showing the migration in the yugabyted UI.
+        {{< /note >}}
 
-    After generating the report, navigate to the **Migrations** tab in the yugabyted UI at <http://127.0.0.1:15433> to see the available migrations.
+    1. After generating the report, navigate to the **Migrations** tab in the yugabyted UI at <http://127.0.0.1:15433> to see the available migrations.
 
-    ![Migration Landing Page](/images/migrate/migration-list-page.png)
-    ![Migration Assessment Page](/images/migrate/ybd-assessment-page.png)
+        ![Migration Landing Page](/images/migrate/migration-list-page.png)
+        ![Migration Assessment Page](/images/migrate/ybd-assessment-page.png)
 
 1. Create a target YugabyteDB cluster as follows:
```

Comment:
This step is redundant - belongs in appropriate section on migration pages.

Reply:
Agreed, but we need to mention what a user can do with the recommendations from the assessment report. I will add it as a tip.